pycbc/strain.py

```diff
@@ -39,7 +39,7 @@
 from scipy.signal import kaiserord
  
 
-next_power_of_2 = lambda n: 2 ** (math.ceil(math.log(n, 2)) + 1)
+next_power_of_2 = lambda n: 1<<(n).bit_length()
```

Comment:
What's the reason for this change? To me this doesn't seem clearer, and speed is not an issue with this function (probably this should be a function, not a lambda).